l10n_it_fatturapa_in/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 AgileBG SAGL <http://www.agilebg.com>
+# Copyright 2015 innoviu Srl <http://www.innoviu.com>
+# Copyright 2018 Lorenzo Battistini
+
+{
+    'name': 'Italian Localization - Fattura Elettronica reception',
+    'version': '10.0.1.0.0',
+    'category': 'Localization/Italy',
+    'summary': 'Electronic invoices reception',
+    'author': 'Agile Business Group, Innoviu, '
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.agilebg.com',
+    'license': 'AGPL-3',
+    "depends": [
+        'l10n_it_fatturapa',
+        'partner_firstname',
+        'l10n_it_withholding_tax_causali',
```

Comment:
direi che questa dipendenza dovrebbe essere sostituita con `l10n_it_causali_pagamento`

Reply:
No, perchè ```l10n_it_withholding_tax_causali``` è il modulo , introdotto anche lui dalla https://github.com/OCA/l10n-italy/pull/576, che dipende da ```l10n_it_withholding_tax``` e ```l10n_it_causali_pagamento``` e che aggiunge  al model ```withholding.tax``` il campo ```causale_pagamento_id```.
Campo poi usato da ```l10n_it_fatturapa_in``` per cercare in odoo la ritenuta indicata nell'XML

Reply:
Hai ragione, non avevo compreso correttamente la funzione del modulo, grazie.